.github/workflows/publish.yaml

```diff
@@ -0,0 +1,15 @@
+name: docker_publish
+
+on:
+  push:
+    branches:
+      - 'f/ci'
+    tags:
+      - 'v*.*.*'
```

Comment:
```suggestion
      - '*'
```
This is sufficient.

Reply:
It's better this way :-), as I don't want to publish on all tags, only version tags.